### ISSUE TITLE
Add reference data mapping for Organizations: Categories

### ISSUE THREAD

**Issue by branchedelac** [FOLIO-FSE/folio_migration_tools#359]
These are contact address categories.

For example:
![image](https://user-images.githubusercontent.com/44287646/192280311-436f5b6c-a022-4b44-a96c-ef6f7c1b9b7d.png)

**Out of scope:**
https://github.com/FOLIO-FSE/folio_migration_tools/issues/412
https://github.com/FOLIO-FSE/folio_migration_tools/issues/411

**Comment by branchedelac** [comment]
Depends on #379.

**Comment by branchedelac** [comment]
Breaking these two out as seaparate issues: 
https://github.com/FOLIO-FSE/folio_migration_tools/issues/412
https://github.com/FOLIO-FSE/folio_migration_tools/issues/411
